components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -332,6 +332,7 @@
 <!-- forcing -->
 <config_use_variable_drag>.false.</config_use_variable_drag>
 <config_use_bulk_wind_stress>.true.</config_use_bulk_wind_stress>
+<config_bulk_wind_stress_interp_isotropic>.false.</config_bulk_wind_stress_interp_isotropic>
```

Comment:
A question for @cbegeman and @vanroekel:  Given the extensive testing in B-cases, should we immediately just switch on isotropic wind stress averaging as the default in master, thus changing, this to `true`.

Reply:
I would support making isotropic default behavior based on your testing

Reply:
I would as well.

Reply:
I agree with changing the default to true

Reply:
Done!  Just left for @cbegeman to approve.